cg/cli/add.py

```diff
@@ -341,8 +341,8 @@ def link_sample_to_case(
 def download_external_delivery_data_to_hpc(context: CGConfig, ticket: str, dry_run: bool):
     """Downloads external data from the delivery server and places it in appropriate folder on
     the HPC"""
-    external_data_api = ExternalDataAPI(config=context)
-    external_data_api.transfer_sample_files_from_source(ticket=ticket, dry_run=dry_run)
+    external_data_api = TransferExternalDataAPI(config=context, ticket=ticket, dry_run=dry_run)
+    external_data_api.transfer_sample_files_from_source()
```

Comment:
I'd prefer it that the ticket was passed here like it used to but stick to whatever you prefer 👍

Reply:
I think this is indeed more in line with what we discussed before. You would be able to instantiate the API once and re-use it with any given ticket. 

If we would require this, i'd say it would be worth sticking to the older way.

Reply:
I also prefer having it not part of the instantiation. Also because I do not find it clear from the name "external_data_api" what ticket is being handled

Reply:
fixed